src/D3MHub.sol

```diff
@@ -221,121 +219,121 @@ contract D3MHub {
 
     // --- Deposit controls ---
     function _wind(bytes32 ilk, ID3MPool _pool, uint256 amount) internal {
-        // IMPORTANT: this function assumes Vat rate of D3M ilks will always be == 1 * RAY (no fees).
-        // That's why this module converts normalized debt (art) to Vat DAI generated with a simple RAY multiplication or division
-        // This module will have an unintended behaviour if rate is changed to some other value.
-        if (amount == 0) {
-            emit Wind(ilk, 0);
-            return;
+        if (amount > 0) {
+            vat.slip(ilk, address(_pool), int256(amount));
+            vat.frob(ilk, address(_pool), address(_pool), address(this), int256(amount), int256(amount));
+            // normalized debt == erc20 DAI (Vat rate for D3M ilks fixed to 1 RAY)
```

Comment:
Can we remove this comment? it visually breaks the nice symmetry with `_unwind`.

Reply:
Yes I saw this one during the walkthrough and was thinking the same. Removed now!